Goobi/src/de/sub/goobi/metadaten/copier/DataCopier.java

```diff
@@ -82,7 +85,14 @@ public DataCopier(String program) throws ConfigurationException {
 	 */
 	public void process(CopierData data) {
 		for (DataCopyrule rule : rules) {
-			rule.apply(data);
+			try {
+				rule.apply(data);
+			} catch (RuntimeException nA) {
+				if (LOG.isInfoEnabled()) {
+					LOG.info("Metadata copy rule »" + rule + "« not applicable for process »" + data.getProcessTitle()
+							+ "«. Rule skipped.");
+				}
```

Comment:
I agree in this case with @matthias-ronge : It is more readable with »« as with "". Compare yourself:

```
Metadata copy rule »/NewspaperYear[*]/NewspaperMonth[*]/NewspaperDay[*]/PeriodicalIssue@ProcessID =format $process.id« not applicable for process »ArtiNo_425390144-1824103102«. Rule skipped.
```

or

```
Metadata copy rule "/NewspaperYear[*]/NewspaperMonth[*]/NewspaperDay[*]/PeriodicalIssue@ProcessID =format $process.id" not applicable for process "ArtiNo_425390144-1824103102". Rule skipped.
```

As I see now one part of copy rules is missing as original copy rule was

```
/NewspaperYear[*]/NewspaperMonth[*]/NewspaperDay[*]/PeriodicalIssue@ProcessID \=format "%1s" $process.id;
```